change/@ni-nimble-components-84b7a395-1b56-4f91-b4bc-7b277151ba58.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It'd be good to summarize the discussion we had at standup about remaining known issues in a comment thread on this PR or the linked issue. (I think the main one was mouse and keyboard respecting the disabled state differently). I think Fred said it wasn't high enough priority to track with a new issue but having a bit of traceability might be helpful if it comes up again.

Reply:
I've updated the PR description to describe the functionality that I am not addressing in this PR (i.e. the keyboard/mouse interactions of disabled parent tree items not being the same).